src/activities/multiplication-tables/MultiplicationTables.qml

```diff
@@ -0,0 +1,254 @@
+            property int rectWidth: main.height/15 //main.width/20
+            property bool answer: true
+        }
+
+        onStart: { Activity.start(items) }
+        onStop: { Activity.stop() }
+
+        /////////////////////////////////////////////////////
+        
+        Grid {
+            id:gridRow
+            anchors {
+                top: parent.top
+                topMargin: 20
+                left: parent.left
+                leftMargin: 20
+            }
+            spacing: items.spacing
+            columns: 11
+            rows: 1
+            Repeater {
+                id: repeaterGridRow
+                model: 11
+                Rectangle {
+                   id: dotsGridRow
+                   color: "white"
+                   width: items.rectWidth
+                   height: items.rectHeight
+                   GCText{ text: index}
+                }
+            }
+
+        }
+
+        Grid {
+            id:gridCol
+            anchors {
+                top: parent.top
+                topMargin: 20
+                left: parent.left
+                leftMargin: 20
+            }
+            spacing: items.spacing
+            columns: 1
+            rows: 11
+            Repeater {
+                id: repeaterGridCol
+                model: 11
+                Rectangle {
+                   id: dotsGridCol
+                   color: "white"
+                   width: items.rectWidth
+                   height: items.rectHeight
+                   GCText{ text: index}
+                }
+            }
+
+        }
+
+        Grid {
+            id: grid
+            anchors {
+                top: gridRow.bottom
+                topMargin: items.spacing
+                left: gridCol.right
+                leftMargin: items.spacing
+            }
+
+            spacing: items.spacing
+            columns: 10
+            rows: 10
+
+            Repeater {
+                id: repeater
+                model: 100
+                Rectangle {
+                    id: dots
+                    width: items.rectWidth
+                    height: items.rectHeight
+                    state: "default"
+                    property bool clickedFlag: false
+                    states:[
+                        State {
+                            name: "default"
+                            PropertyChanges { target: dots; color: "green"}
+                            PropertyChanges { target: dots; clickedFlag: false}
+                        },
+                        State {
+                            name: "active"
+                            PropertyChanges { target: dots; color: "red"}
+                            PropertyChanges { target: dots; clickedFlag: true}
+                        }
+                    ]
+
+                    MouseArea {
+                        anchors.fill: parent
+                        onClicked: {
+                            if (dots.state == "default")
```

Comment:
If I understand your code (sorry, didn't have time to test it yet), you need to press each case to get the good result (which will be quite boring to higher levels)?
I think we can do it more interactive as in the link: the child has to place 2 cards on top and left, we colorize automatically the beams between and the child has to input the result. What do you think?